examples/xgboost/histogram-based/app_site2/config/config_fed_client.json

```diff
@@ -1,33 +0,0 @@
-{
```

Comment:
What about similar changes to app_site1 and app_server subdirectories?

Reply:
I merge them into app_base, and use "data_split_file" now to distinguish/split data for diff. clients